packages/teams-js/src/public/pages.ts

```diff
@@ -66,12 +74,16 @@ export function tabsNavigateToTabHelper(apiVersionTag: string, tabInstance: TabI
   });
 }
 
-export function returnFocusHelper(apiVersionTag: string, navigateForward?: boolean): void {
+export function returnFocusHelper(
```

Comment:
```suggestion
/**
 * @hidden
 */
export function returnFocusHelper(
```

You didn't introduce this but we should probably never have been exporting this and documenting it publicly. Adding a @hidden keyword will prevent docs from getting autogenerated for this function

Reply:
added hidden to restrict it from generating docs